src/relay/pass/fuse_ops.cc

```diff
@@ -678,6 +678,7 @@ class GraphPartitioner {
             } else {
               return (kind <= kBroadcast ||
                       kind == kCommReduce ||
+                      kind == kInjective ||
                       kind == kOutEWiseFusable);
             }
```

Comment:
I think this fix is correct, but in this case we can just simplify this condition to kind <= kOutEwiseFusable
I wonder what is the original reason for leaving out kInjective @tqchen

Reply:
@masahi yes, we can. I wasn’t quite sure if it was intentionally left out. If not, I’ll change to <=

Reply:
This is something I overlooked. However, it is good to leave things as they are because this way it is more clear. 
We also need to add a case to fuse into multiple injective ops, i.e. need to enhance condition in the if branch as well.